### PR TITLE
Fixed modules to now sort by volume number

### PR DESCRIPTION
**Pull request by jaromkmadsen**

### CODE REVIEW

File: lib/konbata.rb [hunk 1]
```diff
@@ -70,6 +70,7 @@ def self.generate_course_structures(entries)
     end
 
     course_structures.each do |_course_code, volumes|
+      course_structures[_course_code] = volumes.sort.to_h
```
Comment (conradbeach) [inline lib/konbata.rb]: Now that `_course_code` is being used, the leading underscore should be removed and it should be `course_code`.

**Review by conradbeach — COMMENTED** [review]
Looks good. I suggested one minor change.

File: lib/konbata.rb (hunk 1 above)
Comment (jaromkmadsen) [reply lib/konbata.rb]: We actually aren't using it just yet but I can still change it if you'd like

Comment (jaromkmadsen) [reply lib/konbata.rb]: Oh wait I see what you're saying, yeah that makes sense

**Review by conradbeach — APPROVED** [review]

**Pull request merged**